src/fetching/putEventResponse.ts

```diff
@@ -0,0 +1,21 @@
+import apiUrl from '../utils/apiUrl';
+
+import { ZetkinEventResponse, ZetkinMembership } from '../types/zetkin';
+
+export default async function putEventResponse(event : number, org : number | undefined) : Promise<ZetkinEventResponse> {
```

Comment:
Same issue here with `undefined`. Also, if `org` is just an ID, it should be named `orgId` (same for `deleteEventResponse()`).